all improvement we have speak about in one pull request

azlux opened the pull request:
Arrangement automatique, background (si tu en veut pas met le en remarque et je ferai un "pull" sans ),changement de quelques tailles dans le css pour les écran 15 pouces et moins (le bouton OK n'était pas accessible :) )

Commit:
all improvement

Commit:
https et script pour pour le contener

azlux commented:
j'ajoute une petit ficher qui s'était pas sychro , je sais pas pourquoi mon client à sauté ce fichier, je vient de le remarquer. (j'ai ajouter https car en cas où ton site est en https tu auras une alerte que tout le contenu n'est pas chiffré)

quent1-fr commented:
Je viens de voir tes modifs et je dois avouer que c'est pas mal du tout (c'est même génial ^^). Concernant l'image de fond, tu pourrais préciser la source et la licence, s'il-te-plaït? Juste histoire de le mettre dans le fichier README.md et de vérifier la compatibilité avec la licence GPL v3.

azlux commented:
source : http://spacekitty.deviantart.com/art/Sunset-Shimmer-Silhouette-Wall-506465387
J'ai demandé à l'auteur pour une diffusion. Mais les images sont en libre téléchargement, je pense que cela ne posera pas de problème.

Commit:
$_GET['token'] error

tu ne peut pas modifier $_GET['token']  car c'est une info venant du client en lecture seule. Le serveur renvoie une erreur à chaque fois.

Commit:
Update interact.php

azlux commented:
Correction d'un bug mineur dans le fichier interact.php

Commit:
correction

Ce n'est pas le $_GET['token'] mais le $_SESSION['token'] qui n'était pas défini. DSL

azlux commented:
J'ai eu la réponse du l'auteur. C'est OK.

> The silhouette wallpapers are under this license:
> http://creativecommons.org/licenses/by-nc-nd/3.0/
> 
> So they may be redistributed as long as it's non-commercial project, provides attribution to me, and is non-derivative. If you've got any questions or concerns, let me know.

Commit:
"!" removed

quent1-fr commented:
Super. Je teste ça ce soir et je merge si je ne trouve aucun problème.

quent1-fr commented:
Je viens de tester les modifications, et ça tourne plutôt bien.
Cependant, j'ai l'impression que le fond orange de l'image colle assez
mal avec le fond bleu (assez moche, soit dit en passant; il faudrait que
je change ça) des pages d'installation et de login. Tu pourrais faire un
commit sans la modification du CSS, mais avec l'image (pour une
intégration future)?

azlux commented:
tu veut que j’enlève que le background ou toutes les modifications effectuées du css ?

quent1-fr commented:
Simplement la ligne incluant le background.

Commit:
images removed for commit

azlux commented:
done

quent1-fr commented:
Simplement la ligne incluant le background.

azlux commented:
heu... oui la ligne est en commentaire (ligne 5 du css) . Ton dernier commantaire est une erreur ou j'ai loupé quelque chose ?

quent1-fr commented:
C'était bien une erreur. Il semblerait que la réponse à l'issue par mail soit partie deux fois.
